### ISSUE TITLE
Textbox cursor focus is not happening sporadically in UWP application

### ISSUE THREAD

**Issue by Pariventhan2015** [microsoft/microsoft-ui-xaml#6884]
We have UWP application and Have Templated Control for TextBox which is extended the TextBox control. 
We are getting an issue sometimes. The cursor is not visible, but able to type,  also not able to capture the Control GotFocus event as well. 

The UWP application is running in KIOSK mode (using shell Launcher V2).

When we got know this issue and trying to remote the system and verifying it, the cursor focus is working good. 

Any suggestion?

**Comment by ojhad** [comment]
@Pariventhan2015 would you be able to provide a minimal repro/steps for this issue?

**Comment by Pariventhan2015** [comment]
> @Pariventhan2015 would you be able to provide a minimal repro/steps for this issue?

1. We have an UWP application, which has more than 200 pages
2. We have customized most of the controls, Textbox, ComboBox, RadioButton, etc..
3. The application has lock concept, when the system is idle for an hour, the software (UI UWP application) got locked automatically.
4. The Lock window is a dialog.
5. So far it was occurred 5 to 6 times, most of the time issue happened after we unlock the application. 
6. We have created custom dialog control (container) which is as part shell page elements and display the pages as dialog, here we were facing one issue, the tab focus is going background when the dialogs are open, since the parent element is in the shell level, so that we are disabling the background controls when any dialog opens and enabling the background controls when the dialogs are closed. We have more suspicious in this area.
6. But sometime it happen when navigate one page another page as well.
7. Also we have Global event to launch the keyboard whenever the TextBox control focused, FocusManager.GotFocus+=.. (but we didn't see any exception in this event method)
